etc/spec/SPECIFICATION.md

```diff
@@ -89,22 +89,23 @@ username.
 If a user wants to keep their username, they have to enter a password by
 editing their user account through the Settings screen.
 
-The Login screen can be either a Simple Login screen or a Password Login
-screen. The Simple Login screen is shown by default.
+## Login
 
-## Simple Login
+The Login screen consists of only a modal window with the heading
+"Ting", a textbox, a button and a password form if that username is registered.
 
-The Simple Login screen consists of only a modal window with the heading
-"Ting", a textbox, a button, and a link.
+![Login screen](http://i.imgur.com/wYUj1ev.jpg)
+*Image: Simple Login*
 
-![Login screen](http://i.imgur.com/FirJeIL.jpg)
-*Image: The simple login screen*
+![Login screen](http://i.imgur.com/xLVm6eV.jpg?1)
+*Image: Password Login*
 
-The placeholder text in the textbox is "Διάλεξε ένα ψευδώνυμο". The textbox is
+The placeholder text in the textbox is "Γράψε ένα ψευδώνυμο". The textbox is
 focused by default when the screen is visited. The placeholder text is hidden
-when the user starts typing some text, unless the text is cleared. Pressing the
-Enter key when the keyboard is focused on the textbox is equivalent to pressing
-the button.
+when the user starts typing some text, unless the text is cleared. In the same
+way works the placeholder for the password box "Γράψε τον κωδικό" that appears
+only if that username is registered. Pressing the Enter key when the keyboard is
```

Comment:
This creates an odd balance which I feel uncomfortable with:

1. If we make the delay too short, then a slow typer will see the form changing as they are typing half of their username.

2. If we make the delay too long, a fast typer will always have to wait that amount before being able to enter their password.

Given these, I don't see how your proposal is beneficial as compared to what we had previously in the spec.